google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -67,6 +68,12 @@
  *     {@code java.log}).
  * <li>{@code com.google.cloud.logging.LoggingHandler.level} specifies the default level for the
  *     handler (defaults to {@code Level.INFO}).
+ *     <p> This configuration also sets the "base severity level".
```

Comment:
I'd prefer this to be taken out of the public javadoc, but I won't block you from submitting it as it is if you feel strongly about keeping it here.